src/pages/admin/main/components/statistics-block/components/statistics-metrics-list/StatisticsMetricsList.module.scss

```
@@ -0,0 +1,109 @@
+@use '@/assets/sass/mixins/typography.mixins' as type;
+
+.list {
+    background: c.$white;
+    border: 1px solid c.$grey-500;
+    border-radius: 8px;
+    padding: 12px;
+}
+
+.header {
+    @include type.small-text-medium;
+    color: c.$grey-900;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)